docs/dsa/algorithms/patternsearchalgorithm.md

```diff
@@ -0,0 +1,69 @@
+---
+id: patternsearching-in-dsa
+title: Pattern Searching Algorithm in Data Structures and Algorithms
+sidebar_label: Pattern Searching Algorithm
+sidebar_position: 1
+description: ""
```

Comment:
`description: ""` is not null so, write description content or may be remove description because null is not good practice

Reply:
resolved